packages/salesforcedx-vscode-core/package.json

```diff
@@ -680,6 +680,12 @@
           ],
           "default": false,
           "description": "%internal_dev_description%"
+        },
+        "salesforcedx-vscode-core.show-org-browser": {
+          "type": [
+            "boolean"
+          ],
+          "default": false
```

Comment:
Will this use a checkbox in the settings? :)

Reply:
This one isn't configured for the checkbox- I thought it would be better to keep it as basic as possible since we don't want customers easily finding it and playing around with it. I'll look into how we can do checkboxes for our other customer-facing flags though!